test/python/default_richards_wells.py

```diff
@@ -348,7 +348,33 @@
 #-----------------------------------------------------------------------------
 # Run and Unload the ParFlow output files
 #-----------------------------------------------------------------------------
-
-dir_name = get_absolute_path('test_output/drich_w')
-mkdir(dir_name)
-drich.run(working_directory=dir_name)
+new_output_dir_name = get_absolute_path('test_output/' + run_name)
```

Comment:
I made a change here to make the output directory the run_name.   originally it was 'drich' which doesn't make as much sense to me.   That's the name of the temp Run object.

Does that make sense to you?   I'm not sure why the Python output was named that way originally.

Reply:
agreed - drich seemed random.